src/routes/safe/components/Transactions/TxsTable/ExpandedTx/OwnersColumn/index.tsx

```diff
@@ -110,7 +111,7 @@ const OwnersColumn = ({
   onTxReject,
   canExecute,
   canExecuteCancel,
-}: ownersColumnProps): React.ReactElement => {
+}: ownersColumnProps): React.ReactElement | null => {
```

Comment:
If this component is rendered, can it return null?

Reply:
Check the render method :)

Reply:
Where it should be returning null?

Checking the component I see that will always render something. And deleting the null doesn't trigger any typescript error